mne/bem.py

```diff
@@ -1359,3 +1361,274 @@ def write_bem_solution(fname, bem):
                                bem['solution'])
         end_block(fid, FIFF.FIFFB_BEM)
         end_file(fid)
+
+
+# #############################################################################
+# Create 3-Layers BEM model from Flash MRI images
+
+def make_flash_bem(subject, subjects_dir, no_flash30=False, no_convert=False,
+                   unwarp=False, overwrite=False, show=False):
+    """Create 3-Layers BEM model from Flash MRI images
+
+    Parameters
+    -----------
+    subject : str
+        Subject name.
+    subjects_dir : str
+        Directory containing subjects data (Freesurfer SUBJECTS_DIR).
+    no_flash30 : bool
+        Skip the 30-degree flip angle data.
+    no_convert : bool
+        Assume that the Flash MRI images have already been converted
+        to mgz files.
```

Comment:
I think that the Python end of these functions should be named consistent with our package, i.e. `flash30=True` and `convert=False`. The *command-line* utility, on the other hand, can maintain compatibility with its shell script equivalent. There will be an inconsistency somewhere, this way it's at the internal conversion we do from cmd line args to Python call, as opposed to Python functions within our own package. As people (hopefully) go to moving Python functions directly, we can have consistently named and presented variables.

Practically it means changing a few booleans and variable names, so the load shouldn't be so bad. @agramfort @jaeilepp WDYT?